[2025-07-07T07:48:22.942+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: ETL_airflare_data_test.transform manual__2025-07-07T07:46:15.296610+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
